### PR TITLE
Init read kinesis status

### PR DESCRIPTION
**Pull request by kadhikari**
For details: https://navitia.atlassian.net/browse/NAV-1693

### CODE REVIEW

File: internal/departures/api.go [hunk 1]
```diff
@@ -14,6 +14,13 @@ type DeparturesResponse struct {
 func DeparturesApiHandler(context *DeparturesContext) gin.HandlerFunc {
 	return func(c *gin.Context) {
 		response := DeparturesResponse{}
+		/*
+		if context.GetStatus() != "ok" {
+			response.Message = context.GetMessage()
+			c.JSON(http.StatusServiceUnavailable, response)
+			return
+		}
+        */
```
Comment (pascal-vs) [inline internal/departures/api.go:23]: forgotten?

File: internal/departures/sirism/sirism.go [hunk 2]
```diff
@@ -204,6 +205,76 @@ func (d *SiriSmContext) processNotificationsLoop(context *departures.DeparturesC
...
+
+			// Update status information
+			var statusText string
+			if statusData.Status {
+				statusText = "ok"
+			} else {
+				statusText = "ko"
+			}
+			// dateLoc, err := time.ParseInLocation("2006-01-02 15:04:05 +0000 UTC", statusData.LastUpdate, d.location)
+			// dateLoc, err := time.ParseInLocation("2006-01-02T15:04:05.999999999+01:00", statusData.LastUpdate, d.location)
```
Comment (pascal-vs) [inline internal/departures/sirism/sirism.go:258]: forgotten?

File: internal/departures/api.go (hunk 1 above)
Comment (kadhikari) [reply internal/departures/api.go:23]: :+1:

File: internal/departures/sirism/sirism.go [hunk 3]
```diff
@@ -253,6 +324,14 @@ func (d *SiriSmContext) RefereshDeparturesLoop(context *departures.DeparturesCon
 	go d.processCleanDeparturesLoop(context)
 }
 
+func (d *SiriSmContext) RefereshStatusLoop(context *departures.DeparturesContext) {
+	// Wait 10 seconds before reloading status notification messages
+	time.Sleep(10 * time.Second)
```
[the remark this answers is not in the document]
Comment (kadhikari) [reply internal/departures/sirism/sirism.go:329]: We don't need sleep here since it's independent.

**Review by pascal-vs — APPROVED** [review]

**Pull request merged**